sycl/source/detail/context_impl.hpp

```diff
@@ -241,9 +243,12 @@ class context_impl {
                        const std::set<std::uintptr_t> &ImgIdentifiers,
                        const std::string &ObjectTypeName);
 
+  bool isOwnedByRuntime() { return MOwnedByRuntime; };
+
   enum PropertySupport { NotSupported = 0, Supported = 1, NotChecked = 2 };
 
 private:
+  bool MOwnedByRuntime;
```

Comment:
I like this name much better than "KeepOwnership" which was always confusing to me.  Kept by whom?    

Thanks!